### ISSUE TITLE
When opening SGF files etc, can the file format be restricted for selection? 

### ISSUE THREAD

**Issue by qcgm1978** [kaorahi/lizgoban#105]
This would make it easier to choose the appropriate files. "Open SGF etc" could be changed to "Open SGF, GIB ...". For example, like this:
```javascript
function select_files(title, dir) {
    const filters = [
        {
          name: 'Go Game Files',
          extensions: ['sgf', 'gib', 'ngf', 'ugf', 'ugi']
        }
      ]
    return dialog.showOpenDialogSync(null, {
        properties: ['openFile'], title: title,
        defaultPath: dir,filters
    }) || []
}
```
<img width="669" alt="截屏2023-12-11 21 07 39" src="https://github.com/kaorahi/lizgoban/assets/3024299/1e66512d-2729-45a0-9f77-8063bd44b998">

**Comment by kaorahi** [comment]
implemented (30cc6baa1). thx for the suggestion!

**Comment by qcgm1978** [comment]
It works well
